lumi/get_slot.sh

```diff
@@ -9,6 +9,10 @@ JENKINS_JAR=$2
 USER=cmsbuild
 SESSION=$OS
 
+# CMS config
+export CMS_SITE_OVERRIDE=T2_FI_HIP
+export ARTIFACTS_USER=cmsbuild
+
 # set up the project number
 export SLURM_ACCOUNT=project_462000245
```

Comment:
I would suggest to not have duplicate of this. e.g. `SLURM_ACCOUNT` from the `connect.sh` script can be passed a command-line argument to `get_slot.sh`